r_examples/r_api_serving_examples/API Serving Examples.ipynb

```diff
@@ -0,0 +1,881 @@
+{
```

Comment:
<p>Please clear all notebook outputs for a cleaner example, and then any future updates will not have a large amount of changed output in the diff.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2902/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>